components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -685,22 +707,17 @@ - (void)setTextInsetsMode:(MDCTextInputTextInsetsMode)textInsetsMode {
 }
 
 - (UITextView *)textView {
-  if (!_textView) {
-    _textView = [[UITextView alloc] initWithFrame:CGRectZero];
-    [self setupTextView];
-    [self setupUnderlineConstraints];
-  }
+  [self textViewSafetyCheck];
   return _textView;
 }
 
 - (void)setTextView:(UITextView *)textView {
   if (![_textView isEqual:textView]) {
     [_textView removeFromSuperview];
     _textView = textView;
-    if (textView) {
-      [self setupTextView];
-      [self setupUnderlineConstraints];
-    }
+    [self textViewSafetyCheck];
```

Comment:
Can we avoid this check here by requiring an MDCIntrinsicHeightTextView as the parameter / property?

Reply:
We could but we were trying to keep that fact private so people just see UITextView in the API instead of a class they have never heard of.